src/Pim/Bundle/DataGridBundle/Normalizer/GroupProductNormalizer.php

```diff
@@ -18,8 +20,12 @@ public function normalize($product, $format = null, array $context = array())
     {
         $data = parent::normalize($product, $format, $context);
 
-        $data['in_group'] = null !== $product->getVariantGroup();
-        $data['is_checked'] = null !== $product->getVariantGroup();
+        $groupIds = array_map(function (GroupInterface $group) {
```

Comment:
No, because it will return a collection. So in the end, I need to perform a "toArray" too.